Source/WebCore/xml/XMLHttpRequest.h

```diff
@@ -54,13 +55,15 @@ class XMLHttpRequestUpload;
 struct OwnedString;
 template<typename> class ExceptionOr;
 
-class XMLHttpRequest final : public ActiveDOMObject, public RefCounted<XMLHttpRequest>, private ThreadableLoaderClient, public XMLHttpRequestEventTarget {
+class XMLHttpRequest final : public ActiveDOMObject, public RefCountedAndCanMakeWeakPtr<XMLHttpRequest>, private ThreadableLoaderClient, public XMLHttpRequestEventTarget {
```

Comment:
This change is not needed since we already subclass `CanMakeWeakPtr` from `ThreadableLoaderClient`

Reply:
Youenn helped me out a bit and I think it's good now.